src/EnergyPlus/DXCoils.cc

```diff
@@ -4623,27 +4705,27 @@ void GetDXCoils(EnergyPlusData &state)
                                                      RoutineName,                     // Routine name
                                                      CurrentModuleObject,             // Object Type
                                                      thisDXCoil.Name,                 // Object Name
-                                                     cAlphaFields(11 + (I - 1) * 6)); // Field Name
+                                                     cAlphaFields(12 + (I - 1) * 6)); // Field Name
 
                 if (!ErrorsFound) {
                     checkCurveIsNormalizedToOne(state,
                                                 std::string{RoutineName} + CurrentModuleObject,
                                                 thisDXCoil.Name,
                                                 thisDXCoil.MSCCapFFlow(I),
-                                                cAlphaFields(11 + (I - 1) * 6),
-                                                Alphas(11 + (I - 1) * 6),
+                                                cAlphaFields(12 + (I - 1) * 6),
+                                                Alphas(12 + (I - 1) * 6),
                                                 1.0);
                 }
             }
 
-            thisDXCoil.MSEIRFTemp(I) = GetCurveIndex(state, Alphas(12 + (I - 1) * 6)); // convert curve name to number
+            thisDXCoil.MSEIRFTemp(I) = GetCurveIndex(state, Alphas(13 + (I - 1) * 6)); // convert curve name to number
             if (thisDXCoil.MSEIRFTemp(I) == 0) {
-                if (lAlphaBlanks(12 + (I - 1) * 6)) {
+                if (lAlphaBlanks(13 + (I - 1) * 6)) {
                     ShowSevereError(state, format("{}{}=\"{}\", missing", RoutineName, CurrentModuleObject, thisDXCoil.Name));
-                    ShowContinueError(state, format("...required {} is blank.", cAlphaFields(12 + (I - 1) * 6)));
+                    ShowContinueError(state, format("...required {} is blank.", cAlphaFields(13 + (I - 1) * 6)));
                 } else {
                     ShowSevereError(state, format("{}{}=\"{}\", invalid", RoutineName, CurrentModuleObject, thisDXCoil.Name));
-                    ShowContinueError(state, format("...not found {}=\"{}\".", cAlphaFields(12 + (I - 1) * 6), Alphas(15 + (I - 1) * 6)));
+                    ShowContinueError(state, format("...not found {}=\"{}\".", cAlphaFields(13 + (I - 1) * 6), Alphas(13 + (I - 1) * 6)));
```

Comment:
Was this a mistake in code and was corrected? I see an alpha index of 12 and 15 before, and now I see 13 and 13. If this fixes a mistake, which it looks like it does, then that should be added to the PR description.

Reply:
It should probably be 13 as the error message was stating what the current value of this field is, so the cAlphaFields and Alphas should have the same index. The 12th (one before) and 14th (one after) fields also have similar formats. So I corrected it.